taketitan/chain.json

```diff
@@ -6,18 +6,18 @@
   "website": "https://taketitan.com/",
   "pretty_name": "TakeTitan",
   "chain_type": "cosmos",
-  "chain_id": "taketitan-12",
-  "bech32_prefix": "taketitan",
-  "daemon_name": "ttnc",
-  "node_home": "$HOME/.ssc",
+  "chain_id": "titan-1",
+  "bech32_prefix": "titan",
+  "daemon_name": "titand",
+  "node_home": "$HOME/.titand",
   "key_algos": [
     "secp256k1"
   ],
-  "slip44": 1179993421,
+  "slip44": 118,
   "staking": {
     "staking_tokens": [
       {
-        "denom": "ttnc"
+        "denom": "utitan"
```

Comment:
An asset with this 'base' must be defined (which currently it's still set to ttnc in the assetlist)